src/renderer/components/theme-settings/theme-settings.js

```diff
@@ -46,7 +46,9 @@ export default defineComponent({
         'catppuccinMocha',
         'dracula',
         'solarizedDark',
```

Comment:
Lets keep this sorted: A-Z

```suggestion
        'gruvboxDark',
        'gruvboxLight'
        'solarizedDark',
        'solarizedLight'
```

Reply:
Looks like the order messed up by my proposed changes. There are duplicates now :(